glib/glib.go

```diff
@@ -256,6 +256,33 @@ func goMarshal(closure *C.GClosure, retValue *C.GValue,
 			} else {
 				val = innerVal
 			}
+
+		case *Variant:
```

Comment:
I this a valid Go syntax? because under go 1.16 it produces this error:
```
# github.com/gotk3/gotk3/glib
../../go/pkg/mod/github.com/gotk3/gotk3@v0.5.3-0.20210305115817-fdbeed35eb7d/glib/glib.go:236:3: expected '}', found 'case'
../../go/pkg/mod/github.com/gotk3/gotk3@v0.5.3-0.20210305115817-fdbeed35eb7d/glib/glib.go:269:2: expected declaration, found rv
```

Reply:
This is indeed not valid syntax. I think I'll make a PR to fix this.

Reply:
It seems like something went wrong while merge conflicts were being solved.

Reply:
Created #743.